src/components/FloatingDropdown/FloatingDropdownBase.tsx

```diff
@@ -55,11 +53,13 @@ export class FloatingDropdownBase extends React.Component<
         <div className={classes.content}>
           <div className={classes.textContent}>
             <span className={classes.label}>{label}</span>
-            <div
-              className={classes.selectedValue}
-              data-hook={DATA_HOOKS.baseSelectedValue}
-            >
-              {selectedOption?.value || placeholder}
+            <div className={classes.selectedValueContainer}>
```

Comment:
I guess we didn't manage to avoid this wrapper... 😬

Reply:
🤔  you know what, maybe we can.
I originally took this approach from here: https://css-tricks.com/flexbox-truncated-text/
But let me check, maybe I can get rid of it